capstone/capdb/tasks.py

```diff
@@ -160,3 +161,35 @@ def get_court_count_for_jur(jurisdiction_id):
     }
 
     return results
+
+
+def create_case_text_for_all_cases(update_existing=False):
+    """
+        iterate through all volumes, call celery task for each volume
+    """
+    query = CaseMetadata.objects.all()
```

Comment:
We can omit duplicative cases here, right?

Reply:
Yep!